src/resources/BreadcrumbResources.ts

```diff
@@ -1,16 +1,29 @@
 import { BreadcrumbIF } from '@/interfaces'
 
-export const RegistryDashboardBreadcrumb: BreadcrumbIF = {
-  text: 'BC Registries Dashboard',
-  href: `${sessionStorage.getItem('REGISTRY_HOME_URL')}dashboard`
+/** Returns URL param string with Account ID if present, else empty string. */
+function getParams (): string {
+  const accountId = sessionStorage.getItem('ACCOUNT_ID')
+  if (accountId) return `?accountid=${accountId}`
+  return ''
 }
 
-export const MyBusinessRegistryBreadcrumb: BreadcrumbIF = {
-  text: 'My Business Registry',
-  href: `${sessionStorage.getItem('BUSINESSES_URL')}business`
+export function getRegistryDashboardBreadcrumb (): BreadcrumbIF {
+  return {
+    text: 'BC Registries Dashboard',
+    href: `${sessionStorage.getItem('REGISTRY_HOME_URL')}dashboard${getParams()}`
+  }
 }
 
-export const StaffDashboardBreadcrumb: BreadcrumbIF = {
-  text: 'Staff Dashboard',
-  href: `${sessionStorage.getItem('BUSINESSES_URL')}staff`
+export function getMyBusinessRegistryBreadcrumb (): BreadcrumbIF {
+  return {
+    text: 'My Business Registry',
+    href: `${sessionStorage.getItem('BUSINESSES_URL')}business${getParams()}`
+  }
+}
+
+export function getStaffDashboardBreadcrumb (): BreadcrumbIF {
+  return {
+    text: 'Staff Dashboard',
+    href: `${sessionStorage.getItem('BUSINESSES_URL')}staff${getParams()}`
+  }
```

Comment:
I had to change these to functions so they are evaluated when called (where ACCOUNT_ID exists), not during compile time (where it doesn't).